src/utils/shared-props.js

```diff
@@ -96,6 +97,7 @@ const PickerDefaultProps = {
   emojiTooltip: EmojiDefaultProps.tooltip,
   autoFocus: false,
   custom: [],
+  iconFns: {},
```

Comment:
/nit I know it's technically an array of function (although I question why) is `iconFns` a little too verbose ? Could just go with `icons` 🤷‍♂️

Reply:
Will change to just `icons`!

Reply:
I actually liked `iconFns`, since it was consistent with `backgroundImageFn` 🤷‍♂️

Reply:
I did consider this before adding it to the review, but noticed that the prop `emojisToShowFilter` is also a function, but isn't called `emojisToShowFilterFn` so went with adding Fn wasn't a necessary suffix

Reply:
Hmmmm fair call. `icons` is fine by me. I suppose eventually we could make it support non-functions too. Like you give it a string and we assume it's a URL to an image or something (just a hypothetical).